application/application.go

```diff
@@ -16,14 +16,14 @@ type Application interface {
 	Trackers() TrackerRepository
 	TrackerQueries() TrackerQueryRepository
 	SearchItems() SearchRepository
-	Identities() IdentityRepository
+	Identities() account.IdentityRepository
 	WorkItemLinkCategories() link.WorkItemLinkCategoryRepository
 	WorkItemLinkTypes() link.WorkItemLinkTypeRepository
 	WorkItemLinks() link.WorkItemLinkRepository
 	Comments() comment.Repository
 	Spaces() space.Repository
 	Iterations() iteration.Repository
-	Users() account.IdentityRepository
+	Users() account.UserRepository
```

Comment:
Hmm.., looking at the code and not so sure now.. We need CRUD operations for both identities and users plus a few specif functions for each of them. Not sure if it gives us any simplicity if we merge them in one interface. We can think about this latter.